### DISCUSSION TITLE
hi tsyuay i know if you played this game as a kid but i wanted you to add this game as my last request, and the game frozen islands. pls!!

### DISCUSSION THREAD

**Discussion by tnt3333334** [Tsyuay/FrivReborn#36] [Q&A] [answered]
this game, wacky wings.

<img width="960" alt="Sem título" src="https://user-images.githubusercontent.com/122381404/213234544-5aeb9b91-b220-4d15-8cd7-ebccead72b6c.png">

swf:https://web.archive.org/web/20170106023412/http://l.funo.com/z/wackywings/wackywings.swf

swf walkthrough:https://web.archive.org/web/20190703192743/http://www.friv.com/x/walkthroughs/wackywings.html

use this links to add the game.

**Comment by tnt3333334** [comment]
put this game, frozen islands.
<img width="960" alt="Sem título" src="https://user-images.githubusercontent.com/122381404/213309317-70642bdf-2862-4f96-9d91-f995c0b1541f.png">

**Comment by Tsyuay** [comment, marked as answer]
done
